VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-toastr.service.ts

```diff
@@ -432,6 +432,55 @@ export class NotificationToastrService {
         return toast.toastRef.componentInstance as VhToastComponent;
     }
 
+    createAllocationNotificationToast(hearings: any[]): VhToastComponent {
+        const toast = this.toastr.show('', '', {
+            timeOut: 0,
+            extendedTimeOut: 0,
+            toastClass: 'vh-no-pointer',
+            tapToDismiss: false,
+            toastComponent: VhToastComponent
+        });
+
+        const header = `<div class="govuk-!-font-weight-bold toast-content toast-header">${this.translateService.instant(
+            'allocations-toastr.header'
+        )}</div></br></br>`;
+
+        let messageBody = '';
+
+        hearings.forEach(h => {
+            const judge = h.judge;
+            const time = h.time;
+            const caseName = h.case_name;
+
+            messageBody += '<div class="govuk-!-font-weight-bold">' + time + '</div>';
+            messageBody += '<div class="govuk-!-font-weight-bold">' + judge + '</div>';
+            messageBody += '<div class="govuk-!-font-weight-bold">' + caseName + '</div>';
+            messageBody += '</br></br>';
```

Comment:
I recommend wrapping the 3 divs above in 1 div and adding a margin to this to acheive spacing, as opposed to the line breaks.

Reply:
Agreed to leave as-is for now, as the same approach is taken many places in the file